config/routes.rb

```diff
@@ -1,3 +1,19 @@
 Rails.application.routes.draw do
+  # get '/invite', to: 'invitations/new'
+  get 'invitations/show'
```

Comment:
What's this for? Show route is already covered in line 17, and you don't have a show controller either. Check `$ rails routes | grep invitation` and notice there are two routes for the `show` action.

In addition, you can limit the actions as:
`resources :invitations, only: [:index, :show, :new]`

Try this approach when defining routes unless there's need to be more specific. 

[`REQUIRED`]